src/global.ts

```diff
@@ -1,3 +1,21 @@
 export { Auth0Plugin } from './plugin';
 export * from './interfaces';
 export * from './guard';
+
+export {
+    AuthorizationParams,
+    PopupLoginOptions,
+    PopupConfigOptions,
+    GetTokenWithPopupOptions,
+    LogoutOptions,
+    LogoutUrlOptions,
+    CacheLocation,
+    GetTokenSilentlyOptions,
+    IdToken,
+    User,
+    ICache,
+    InMemoryCache,
+    LocalStorageCache,
+    Cacheable,
+    RedirectLoginOptions,
+  } from '@auth0/auth0-spa-js';
```

Comment:
Should we patch `RedirectLoginOptions` and `LogoutOptions` to Omit `onRedirect` ?

If so, we will need to ensure we update `./interfaces/auth0-vue-client.ts` and `plugin.ts` to use our internal types that Omit onRedirect.